prow/github/client.go

```diff
@@ -1022,6 +1024,40 @@ func (c *client) CreateRepoHook(org, repo string, req HookRequest) (int, error)
 	return c.createHook(org, &repo, req)
 }
 
+func (c *client) deleteHook(org string, repo *string, id int, req HookRequest) error {
+	if c.dry {
+		return nil
+	}
+	var path string
+	if repo != nil {
+		path = fmt.Sprintf("/repos/%s/%s/hooks/%d", org, *repo, id)
+	} else {
+		path = fmt.Sprintf("/orgs/%s/hooks/%d", org, id)
+	}
+
+	_, err := c.request(&request{
+		method:      http.MethodDelete,
+		path:        path,
+		exitCodes:   []int{204},
+		requestBody: &req,
```

Comment:
Are you sure a request body is needed? The documentation doesn't mention it and I don't know why it would be needed. https://developer.github.com/v3/repos/hooks/#delete-a-hook

Reply:
Removed. Thanks for catching this!